### ISSUE TITLE
Site reports "Yes" even without WiFi

### ISSUE THREAD

**Issue by aengelke** [kammt/doesmywifiwork#1]
On wired LAN and WAN connections, the site erroneously reports "Yes". Expected: "No".

**Comment by jmrtsh** [comment]
Can reproduce. I think this could be solved using deep neural blockchains.

**Comment by kammt** [comment]
I'll reopen this, since GitHub automatically closed it due to the PR title (looking at you Jim).
`navigator.connection` does not work on e.g. Safari and Firefox. These browsers are now greeted with a big fat "Maybe."

**Comment by jmrtsh** [comment]
Yes, as the PR title indicates, I didn't intend to close this yet, GitHub seems a bit overeager in its natural language processing here.

**Comment by jmrtsh** [comment]
Just wanted to point out, that `navigator.connection` isn't only not supported in Firefox, support for it has been removed after it was previously added. I.e. imo there is no hope of it being added back in in the future.
So maybe we do need to fall back to the original deep neural blockchains approachs after all...
